src/test/java/edu/kis/powp/jobs2d/TestJobs2dPatterns.java

```diff
@@ -30,6 +31,21 @@ private static void setupPresetTests(Application application) {
 
 		application.addTest("Figure Joe 1", selectTestFigureOptionListener);
 		application.addTest("Figure Joe 2", selectTestFigureOptionListener);
+		application.addTest("Rectangle", new SelectTestFigureOptionListener(DriverFeature.getDriverManager())
+		{
+			@Override
+			public void actionPerformed(ActionEvent e) {
+				ComplexCommandFactory.createRectangle(DriverFeature.getDriverManager().getCurrentDriver(), 0, 0, 200, 100).execute();
+			}
+		});
+
+		application.addTest("Triangle", new SelectTestFigureOptionListener(DriverFeature.getDriverManager())
+		{
+			@Override
+			public void actionPerformed(ActionEvent e) {
+				ComplexCommandFactory.createTriangle(DriverFeature.getDriverManager().getCurrentDriver(), 0, 0, 100, 100).execute();
```

Comment:
It is good to keep a convention used in the original code (class instance instead of lambda)